src/styles/theme.scss

```diff
@@ -207,10 +249,6 @@ h5 {
 
 ------------------------------ */
 
-.section {
-  background-color: $white;
```

Comment:
why remove?

Reply:
because Holly's design left these out :-) but I can add these back anytime (I kind of liked it with those in).